### ISSUE TITLE
[MAIN] [Pipeline] [Model1] Configure training setup

### ISSUE THREAD

**Issue by atalantayang** [Mars-Martiny/CITS5206_group1#54]
Set up the training configuration and connect all modules in main_notebook for model1.

Configure training config dict (model, dataloaders, criterion, optimiser, scheduler, epochs, save_dir)
Connect preprocessing, tokenization, encoding, data loading, embedding, and classifier

**Comment by Mars-Martiny** [comment]
Fill in the following details assignee and milestone, also state PR in which this issue was closed.

**Comment by atalantayang** [comment]
This was something I added for model1 before we had assigned issues. Since we hadn't reached the model1 stage yet and were still in the exploration/discussion phase, I closed these two issues. The tasks were then assigned to us last week based on the spreadsheet.
